### ISSUE TITLE
When update app_links from 4.0.1 to ^5.0.0 "Listen to media sharing coming from outside the app while the app is in the memory" not work

### ISSUE THREAD

**Issue by bigzhu** [KasemJaffer/receive_sharing_intent#294]
When update app_links from 4.0.1 to ^5.0.0

iOS

* `Listen to media sharing coming from outside the app while the app is in the memory` not work
* `Get the media sharing coming from outside the app while the app is closed` work fine

**Comment by vontdeux** [comment]
yes, this still exists in latest version of code, it will keep opening new instances of the app
